crates/nu-cmd-extra/src/extra/bytes/mod.rs

```diff
@@ -1,15 +1,15 @@
-mod add;
-mod at;
-mod build_;
-mod bytes_;
-mod collect;
-mod ends_with;
-mod index_of;
-mod length;
-mod remove;
-mod replace;
-mod reverse;
-mod starts_with;
+pub(crate) mod add;
+pub(crate) mod at;
+pub(crate) mod build_;
+pub(crate) mod bytes_;
+pub(crate) mod collect;
+pub(crate) mod ends_with;
+pub(crate) mod index_of;
+pub(crate) mod length;
+pub(crate) mod remove;
+pub(crate) mod replace;
+pub(crate) mod reverse;
+pub(crate) mod starts_with;
```

Comment:
What is the need to make this blanket visible? There have been recently a number of compiler warnings due to potential namespace conflicts due to many similar named symbols imported via `use module::*` that arise from symbols being public.

Reply:
@sholderbach thanks great catch ! I made your requested changes....